### PR TITLE
Convert to setup.cfg / PEP 517

### PR DESCRIPTION
**Pull request by MetRonnie**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[build-system]
+# NOTE: `pip install build` to build with `python -m build`
+requires = [
+  "setuptools >= 40.9.0",
+  "wheel"
+]
+build-backend = "setuptools.build_meta"
```
Comment (MetRonnie) [inline pyproject.toml:7]: Actually, because we're not publishing this as a package, do we even need `pyproject.toml`?

Comment (oliver-sanders) [reply pyproject.toml:7]: No.

**Review by oliver-sanders — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (MetRonnie) [reply pyproject.toml]: Deleted it and force-pushed. (Had a go building the project beforehand and it built successfully, to lend credibility to the setup.cfg implementation)

Comment (MetRonnie) [reply pyproject.toml]: Further update: looks like we don't need any _content_ in `pyproject.toml` if we're not publishing the package, so force pushed empty `pyproject.toml`

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**